.github/workflows/release-tagging.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Tagging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tagging:
+    runs-on: ubuntu-latest
+    env:
+      EXCLUDED_PACKAGE_OA_RENDERERS: '!./packages/vckit-oa-renderers/'
+      EXCLUDED_PACKAGE_CREDENTIAL_OA: '!./packages/credential-oa/'
+      EXCLUDED_PACKAGE_EXPLORER: '!./packages/demo-explorer/'
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Install pnpm
```

Comment:
We're using yarn with `yarn.lock`. Should we install `pnpm`? 
<img width="205" alt="image" src="https://github.com/uncefact/tests-untp/assets/59736798/c56a4718-40ac-45fb-b2d2-b05b6ffca4a5">